pom.xml

```diff
@@ -188,8 +188,8 @@ under the License.
       <dependency>
         <groupId>com.google.inject</groupId>
         <artifactId>guice</artifactId>
-        <version>4.2.3</version>
-        <classifier>no_aop</classifier>
+        <version>6.0.0</version>
+        <scope>test</scope>
```

Comment:
How does this relate to Plexus Testing?

Reply:
Plexus testing uses it. With the old version -  tests will fail

Reply:
Alright, I see. Please leave a comment in the commit message so that this information does not get lost.